3rdparty/openexr/IlmImf/ImfAttribute.cpp

```diff
@@ -81,28 +49,17 @@ class LockedTypeMap: public TypeMap
 {
   public:
 
-    Mutex mutex;
+#if ILMTHREAD_THREADING_ENABLED
+    std::mutex mutex;
+#endif
```

Comment:
It might be a good idea to force `ILMTHREAD_THREADING_ENABLED` to `1` somewhere. Or at least make it the default. Otherwise it may be too easy for library users to shoot themselves in the foot.

Reply:
Done.